requirements.txt

```diff
@@ -3,6 +3,7 @@ replicate==0.8.1
 pytest==7.3.0
 async-timeout==4.0.2
 attrs==22.2.0
+beautifulsoup4==4.12.2
```

Comment:
Please remove beautifulsoup if it is not using. 
To: @jupiter0913

Reply:
I think it should be remain because it is used in email plugin.